app/assets/v2/js/pages/bounty_details.js

```diff
@@ -621,6 +645,10 @@ var build_detail_page = function(result) {
     }
   }
 
+  $('body').delegate('#bounty_details .button.disabled', 'click', function(e) {
```

Comment:
Maybe can be more secure to use an specific class for these buttons that will be disabled because #bounty_details is a big div with many tags inside and any link with class '.button' added inside this div in some situation will be unexpected disabled.